src/systemc/ext/core/sc_port.hh

```diff
@@ -126,7 +126,7 @@ class sc_port_b : public sc_port_base
  * code is correct).
  * Please check section 9.3 of SystemC 2.3.1 release note for more details.
  */
-#if defined(__GNUC__) && (__GNUC__ >= 13)
+#if defined(__clang__) || (defined(__GNUC__) && (__GNUC__ >= 13))
 #pragma GCC diagnostic ignored "-Woverloaded-virtual"
```

Comment:
Does clang understand this pragma? I had assumed `GCC diagnostic ignored` was only understood by GCC?

Reply:
Yes, clang tries to stay compatible with GCC pragrams, so it implements most of them.